### PR TITLE
Bunch of small fixes

### PR DESCRIPTION
**Pull request by thebenkogan**

### CODE REVIEW

File: readme-sync/v0/senseml-reference/3000 - natural-language-methods/1000 - list.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Parameters
 
 | key                        | value  | description                                                  |
 | :------------------------- | :----- | :----------------------------------------------------------- |
-| id (**required**)          | `list` | The Anchor parameter is optional for fields that use the Question method. If you specify an anchor, Sensible ignores it. |
+| id (**required**)          | `list` | The Anchor parameter is optional for fields that use the List method. If you specify an anchor, Sensible ignores it. |
```
Comment (fscelliott) [inline readme-sync/v0/senseml-reference/3000 - natural-language-methods/1000 - list.md:44]: maybe just change to "this method"

File: readme-sync/v0/senseml-reference/3000 - natural-language-methods/1000 - nlp-table.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Parameters
 
 | key                    | value      | description                                                  |
 | :--------------------- | :--------- | :----------------------------------------------------------- |
-| id (**required**)      | `nlpTable` | The Anchor parameter is optional for fields that use the Question method. If you specify an anchor, Sensible ignores it. |
+| id (**required**)      | `nlpTable` | The Anchor parameter is optional for fields that use the NLP Table method. If you specify an anchor, Sensible ignores it. |
```
Comment (fscelliott) [inline readme-sync/v0/senseml-reference/3000 - natural-language-methods/1000 - nlp-table.md:41]: "this method"

File: readme-sync/v0/senseml-reference/3000 - natural-language-methods/1600 - summarizer.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ The following parameters are in the computed field's [global Method](doc:compute
 
 | key                      | value        | description                                                  |
 | :----------------------- | :----------- | :----------------------------------------------------------- |
-| id (**required**)        | `summarizer` | The Anchor parameter is optional for fields that use this method. If you omit an anchor, Sensible searches the entire document for the data you want to extract. |
+| id (**required**)        | `summarizer` | The Anchor parameter is optional for fields that use the Summarizer method. If you omit an anchor, Sensible searches the entire document for the data you want to extract. |
```
Comment (fscelliott) [inline readme-sync/v0/senseml-reference/3000 - natural-language-methods/1600 - summarizer.md:22]: yeah I'd keep it at "this method"

File: readme-sync/v0/senseml-guides/3000 - concepts/3000 - lines.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ When text such as handwriting is slightly misaligned vertically, line sorting is
 
 ![Click to enlarge](https://raw.githubusercontent.com/sensible-hq/sensible-docs/main/readme-sync/assets/v0/images/final/line_sort_2.png)
 
-To configure this default sorting behavior, see [the XMajorSort parameter](doc:method).
+To configure this default sorting behavior, see [the sortLines parameter](doc:method).
```
[the remark this answers is not in the document]
Comment (thebenkogan) [reply readme-sync/v0/senseml-guides/3000 - concepts/3000 - lines.md]: Fixed

File: readme-sync/v0/senseml-reference/3000 - natural-language-methods/1000 - list.md (hunk 1 above)
Comment (thebenkogan) [reply readme-sync/v0/senseml-reference/3000 - natural-language-methods/1000 - list.md]: Fixed this and all other ones

**Pull request merged**